apps/frontend/web/src/components/popups/EventModal.tsx

```diff
@@ -0,0 +1,128 @@
+import {
+    Badge,
+    Box,
+    Button,
+    Center,
+    Flex,
+    Heading,
+    Image,
+    Modal,
+    ModalBody,
+    ModalCloseButton,
+    ModalContent,
+    Text,
+    useDisclosure,
+  } from '@chakra-ui/react';
+  
+  function EventModal() {
+    const { isOpen, onOpen, onClose } = useDisclosure();
+    return (
+      <Box>
+        <Button onClick={onOpen}>{'Open Modal'}</Button>
+  
+        <Modal isOpen={isOpen} onClose={onClose} size={'full'}>
+          <ModalContent bg={'yellow.200'}>
+            <ModalCloseButton />
+            <ModalBody pt={'10'} px={'0'}>
+              <Box h={'calc(100vh-7rem)'}>
+                <Center>
+                  <Image
+                    boxSize="35%"
+                    m="auto"
+                    mt="2.5"
+                    src="/hackathon.png"
+                    alt="hackathon image"
+                  />
+                </Center>
+                <Center>
+                  <Heading size="md" py={'10'}>
+                    {'Hackaton'}
+                  </Heading>
+                </Center>
+                <Text textAlign={'center'}>
+                  {
+                    'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Veritatis maxime fugiat pariatur aliquid, ut ratione neque? Quamlaudantium ipsa velit, placeat reiciendis rerum, totam quo dictaexpedita itaque ea eius?'
+                  }
+                </Text>
+                <Center>
+                  <Flex py={'10'}>
+                    <Text mr={'3'} fontWeight={'extrabold'}>
+                      {'Current price :'}
```

Comment:
Pas d'espace avant `:`
```suggestion
                      {'Current price:'}
```